content/en/data_observability/integrations/business_intelligence/tableau.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Tableau
+description: Connect Tableau to Datadog Data Observability to view end-to-end lineage from warehouse tables to dashboards.
+further_reading:
+  - link: '/data_observability/data_quality'
+    tag: 'Documentation'
+    text: 'Learn about Data Quality monitoring'
+---
+
+## Overview
+
+Datadog's Tableau integration helps data teams make changes to their data platform without breaking dashboards, and identify unused workbooks and data sources to remove. When Datadog connects, it:
+
+- Pulls metadata from your Tableau site like fields, worksheets, dashboards, workbooks, and data sources
+- Automatically generates lineage between warehouse tables/columns with downstream Tableau fields, worksheets, dashboards, and workbooks.
+
+## Connect Tableau
+
+### Tableau requirements
+
+In order for Datadog to extract your metadata from Tableau you must meet all of the [Tableau Metadata GraphQL][1] requirements:
+
+- Tableau Cloud/Server v2019.3+
+- Tableau REST API must not be disabled
```

Comment:
Can we say "must be enabled" here? I understand if there are technical limitations around it, but the double negative is a little hard to parse.